openassessment/xblock/static/dist/manifest.json

```diff
@@ -4,8 +4,8 @@
   "openassessment-editor-textarea.js.map": "/openassessment-editor-textarea.e855a75d3004c5ff54ee.js.map",
   "openassessment-editor-tinymce.js": "/openassessment-editor-tinymce.f7c8d3bff424e0efaeee.js",
   "openassessment-editor-tinymce.js.map": "/openassessment-editor-tinymce.f7c8d3bff424e0efaeee.js.map",
-  "openassessment-lms.js": "/openassessment-lms.aba4112d7bf75980425a.js",
-  "openassessment-lms.js.map": "/openassessment-lms.aba4112d7bf75980425a.js.map",
+  "openassessment-lms.js": "/openassessment-lms.41ca7aa5d2056aa114a5.js",
```

Comment:
did the `js` files for this got change? if it did, I am not seeing part of the commit.